### PR TITLE
Simplified wizard step 3 and 5 accordingly

### LINKED ISSUE #5013: Wizard Form: Simplify design
Please simplify the design of the wizard as follows:

Wizard Step 1
* Simplify page by taking out 4 horizontal lines as shown below

screencapture-open-event-frontend-git-fork-iamareebjamal-online-events-eventyay-vercel-app-create-2020-09-08-10_07_32

Wizard Step 3
* Add Custom Form Field - demote from heading to standard text. This should not be a heading.

Screenshot from 2020-09-08 10-37-59

Wizard Step 5
* "Customize registration form" rename to -> "Customize submission forms"
* Add Custom Form Field - demote from heading to standard text. This should not be a heading.

Screenshot from 2020-09-08 10-39-11

### PR DESCRIPTION
**Pull request by terminator0309**
Fixes #5013 

#### Short description of what this resolves:
Simplifies wizard Step 3 and 5

#### Changes proposed in this pull request:
1. Wizard step 3 : demoted "Add Custom form field" from heading to standard text.
2. Wizard step 5 : "Customize registration form" renamed to -> "Customize submission forms"

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by mariobehling — CHANGES_REQUESTED** [review]
Thanks! Please resolve conflicts.

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 1]
```diff
@@ -19,213 +20,201 @@
...
-                </button>
-              {{/if}}
-              <button class="ui icon primary button" type="button" {{action 'addItem' 'microlocation'}}>
-                <i class="plus icon"></i>
-              </button>
-            </div>
-          </div>
-        </div>
-      {{/each}}
+  <br>
```
Comment (iamareebjamal) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs:23]: Please don't change indentation unnecessarily. I can't see what's your change and what's just indentation change. Revert to original indentation and just make your changes in the PR

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Unnecessary indentation changes

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs (hunk 1 above)
Comment (terminator0309) [reply app/templates/components/forms/wizard/sessions-speakers-step.hbs:23]: Sorry for that, actually i use Prettier and as soon as i saved the changes it formatted everything. I will make a new PR.

Comment (iamareebjamal) [reply app/templates/components/forms/wizard/sessions-speakers-step.hbs:23]: No need for a new PR, push changes to this PR itself

**Pull request closed without merging**